### ISSUE TITLE
Project status? Doesn't seem maintained

### ISSUE THREAD

**Issue by Vadorequest** [klaussinani/signale#108]
It seems like this package isn't maintained anymore. Am I mistaken?

A few PR are hanging, nothing useful has been merged since 2018.

**Comment by rudemex** [comment]
Hi @Vadorequest, I solved PR, such as formatting the day and time and configuring the time zone and adjusting the log levels, also adding the display of the line and column that makes the log, all configurable.

https://www.npmjs.com/package/signale-logger

**Comment by davidkhala** [comment]
> Hi @Vadorequest, I solved PR, such as formatting the day and time and configuring the time zone and adjusting the log levels, also adding the display of the line and column that makes the log, all configurable.
> 
> https://www.npmjs.com/package/signale-logger

It is a brand-new one? I could not see github fork subtitle in the repos.

**Comment by Vadorequest** [comment]
Looking at https://www.npmjs.com/package/signale-logger versions, it does seem like a whole new package on NPM.

**Comment by rudemex** [comment]
Hello @Vadorequest @davidkhala , signale-logger is the last version of signale (1.4.0) to which I applied some PR that requested and were on hold, and as I had an urgency to use it I decided to upload it to npm, when they do the PR of the earrings (in signale) , I estimate that this package I will deprecate it.

**Comment by Vadorequest** [comment]
Thanks for the clarification!

**Comment by jakobrosenberg** [comment]
I'm on the fence about using this. Is there any indication that this library is still maintained?

**Comment by rudemex** [comment]
hello @jakobrosenberg , the original library has not been updated for a long time and has some PR pending, if you want, you can use the fork I made from the original and implement some improvements they requested.

https://www.npmjs.com/package/signale-logger

**Comment by jakobrosenberg** [comment]
@rudemex thanks for maintaining a fork.

I wish the author would assign a maintainer, release a statement or reply to this issue.

**Comment by stephanebachelier** [comment]
@rudemex maybe we should create a public organization to host a fork and enable more contributors to join.
We could create a `signale` github org and use the `@signale` npm fork (if available).

**Comment by rudemex** [comment]
hi @stephanebachelier , how are you? i created a project that has available to implement PR following the requirements of the original project, it is also deployed in npm and yarn, i don't think it is necessary to create an organization, and the project with signale is occupied by the original.
I'm following up the project, I have some pending tasks to solve, but PR is welcome.

Repo Signale-logger: [https://github.com/rudemex/signale-logger](https://github.com/rudemex/signale-logger)

**Comment by stephanebachelier** [comment]
Hi @rudemex,

The idea is not mine. I've already seen it in other open-source repositories when the maintainer was missing. 
Some contributors and/or users have decided to create an organisation to open the project to a community of contributors to avoid the « single contributor syndrome ». If the fork is maintained by a single contributor, who might not available or missing at some point, - which might be expected on a long term basis -, the same problem surfaces again.

IMO, an open organisation, such as a Github org, seems more appropriated.

**Comment by rudemex** [comment]
@stephanebachelier yes, it's not a bad idea, but I think it will generate conflict to have many dependencies created in npm on the same project, my repo is public and I can add collaborators, so you can give a maintenance to the project.
If you think so, I'll add you as a collaborator.

**Comment by hustcer** [comment]
Is this repo still alive?

**Comment by binyamin** [comment]
@hustcer After reading through this issue, it seems there's a community fork at https://github.com/rudemex/signale-logger (h/t @rudemex)

**Comment by JCMais** [comment]
I've been using https://github.com/unjs/consola nowadays
